### ISSUE TITLE
Northstar's Dropdown contrast ratio is less than 3:1 

### ISSUE THREAD

**Issue by davyrwoo** [microsoft/fluentui#12505]
### Environment Information

- **Package version(s)**: @fluentui/react-northstar@0.47.0

### Describe the issue:

The color contrast ratio is less than 3:1 for the dropdown field in light and dark themes. 

### Please provide a reproduction of the issue in a codepen:

https://codesandbox.io/s/jbglf?module=/example.js

#### Actual behavior:

Contrast ratio for the dropdown component is less than 3:1

#### Expected behavior:

The component color or the boundary/border color of the component should have a contrast ratio of at least 3:1

### Documentation describing expected behavior

https://www.w3.org/TR/WCAG21/#non-text-contrast

**Comment by jurokapsiar** [comment]
Tracking Reference: 7402 (used for team planning)

**Comment by assuncaocharles** [comment]
It's fixed in the current master version.
